src/components/assets/svg/index.js

```diff
@@ -0,0 +1,6 @@
+export { Notesicon } from "./notesIcon";
+export { Archiveicon } from "./archiveIcon";
+export { Trashicon } from "./trashIcon";
+export { Loginicon } from "./loginIcon"
+export { Coloricon } from "./colorIcon"
+export {Tagicon} from "./tagIcon"
```

Comment:
You could have used package such as `react-icons` instead of copying and exporting SVGs